miniapps/navier/CMakeLists.txt

```
@@ -53,4 +53,11 @@ if (MFEM_USE_MPI)
     MAIN navier_3dfoc.cpp
     ${NAVIER_COMMON_FILES}
     LIBRARIES mfem)
+
+  if (MFEM_USE_GSLIB)
+    add_mfem_miniapp(navier_cht
+      MAIN navier_cht.cpp
+      ${NAVIER_COMMON_FILES}
+      LIBRARIES mfem)
+  endif ()
```

This should fix the issue related to the mesh files not being copied over for the navier_cht executable when using cmake.
```suggestion

    LIST(APPEND NAVIER_MESH_FILES
        fluid-cht.mesh
        solid-cht.mesh)
    foreach(MESH_FILE ${NAVIER_MESH_FILES})
        add_custom_command(
          TARGET navier_cht POST_BUILD #OUTPUT ${MESH_FILE}
          COMMAND ${CMAKE_COMMAND} -E copy_if_different
          ${CMAKE_CURRENT_SOURCE_DIR}/${MESH_FILE} ${MESH_FILE}
          COMMENT "copy: ${MESH_FILE}")
    endforeach()
  endif ()
```